apps/web/pages/_app.tsx

```diff
@@ -21,6 +23,13 @@ import I18nLanguageHandler from "@components/I18nLanguageHandler";
 
 import "../styles/globals.css";
 
+const interFont = Inter({ subsets: ["latin"], variable: "--font-inter", preload: true });
+const calFont = localFont({ src: "../fonts/CalSans-SemiBold.woff2", variable: "--font-cal", preload: true });
+
+if (typeof document !== "undefined") {
```

Comment:
a workaround for font not applied in modals and tooltips.

Reply:
how come it doesnt work in modals and tooltips?

Reply:
next font works by optimizing the font at build time and assigning a hash to it. in order to work we have assign the hash to the `css variable` we provided. if we follow the same step as the [docs](https://nextjs.org/docs/basic-features/font-optimization#with-tailwind-css), `css variable` will be asssigned inside a container with an id `__next`. whereas modals render outside this container, so it cannot inherit the assigned value. assigning the variable at body seems to fix it.

Reply:
Moved this into `:root` inspired by https://larsmagnus.co/blog/how-to-optimize-custom-fonts-with-next-font - this is also in the next docs (but maybe wasn't there a month ago)